src/openrct2/object/TerrainEdgeObject.cpp

```diff
@@ -58,3 +60,15 @@ void TerrainEdgeObject::ReadJson(IReadObjectContext* context, json_t& root)
     PopulateTablesFromJson(context, root);
     NumImagesLoaded = GetImageTable().GetCount();
 }
+
+TerrainEdgeObject* TerrainEdgeObject::GetById(ObjectEntryIndex entryIndex)
+{
+    TerrainEdgeObject* result = nullptr;
+    auto& objMgr = OpenRCT2::GetContext()->GetObjectManager();
+    auto obj = objMgr.GetLoadedObject(ObjectType::TerrainSurface, entryIndex);
+    if (obj != nullptr)
+    {
+        result = static_cast<TerrainEdgeObject*>(obj);
+    }
+    return result;
```

Comment:
You can make this:
```suggestion
    return obj != nullptr ? static_cast<TerrainEdgeObject*>(obj) : nullptr;
```

Then you wouldn't need the `result` variable.